Dockerfile.arm.release

```diff
@@ -23,8 +23,7 @@ RUN \
      chmod +x /usr/bin/minio && \
      chmod +x /usr/bin/docker-entrypoint.sh && \
      chmod +x /usr/bin/verify-minio.sh && \
-     curl -s -q -O https://raw.githubusercontent.com/minio/minio/master/CREDITS && \
-     /usr/bin/verify-minio.sh
```

Comment:
Why are we removing this? `shasum` binary verification is not needed anymore?

Reply:
arm32 - minsign doesn't work anymore on alpine